compose/compose-file.md

```diff
@@ -1409,6 +1410,29 @@ refer to it within the Compose file:
         external:
           name: actual-name-of-network
 
+## Extra configuration reference
```

Comment:
@londoncalling the PR which adds this feature has not merged, so no docs changes are required.